### ISSUE TITLE
[Enhancement] Cross-platform image load/save APIs

### ISSUE THREAD

**Issue by davidbritch** [dotnet/Microsoft.Maui.Graphics#141]
While it's possible to use `GraphicsService.LoadImageFromStream` to load images from a variety of sources (local files, embedded files etc.), it seems to involve writing platform code if the image isn't an embedded resource. Ditto for `IImage.Save`.

For MAUI usage, a fully cross-platform image load/save API would be beneficial so that you could load/save an image from/to a known location, without having to resort to platform code.

**Comment by charlesroddie** [comment]
> While it's possible to use GraphicsService.LoadImageFromStream to load images from a variety of sources (local files, embedded files etc.), it seems to involve writing platform code

So you have a valid stream but it doesn't work. So this is a bug report about GraphicsService.LoadImageFromStream?

**Comment by jonlipsky** [comment]
@davidbritch I'm not sure what the issue/ask is.  Loading images from a Stream seems (to me) the most cross platform approach to take.  Are you looking for extension methods that hides file management as well?

**Comment by davidbritch** [comment]
Yes. I'm just looking at it purely from a perspective of usage in MAUI.

**Comment by charlesroddie** [comment]
Post in https://github.com/dotnet/maui instead? The similarity in naming of the repos is unfortunate!
